tests/dm.go

```diff
@@ -60,7 +60,7 @@ const (
 	// DMMySQLSvcStsName is the upstream MySQL svc/sts name for DM E2E tests.
 	DMMySQLSvcStsName = "dm-mysql"
 	// DMMySQLImage is the upstream MySQL container image for DM E2E tests.
-	DMMySQLImage = "mysql:5.7"
+	DMMySQLImage = "mysql:8.0"
```

Comment:
in MySQL 5.7, Golang (v1.22) client will get "remote error: tls: handshake failure"

ref https://github.com/elastic/beats/pull/40082#issuecomment-2227716350